vaadin-upload.html

```diff
@@ -279,6 +283,17 @@
       },
 
       /**
+       * Specifies if the maximum number of files have been uploaded
+       */
+      hasMaxFiles: {
+        type: Boolean,
+        value: false,
+        notify: true,
+        readOnly: true,
+        computed: '_maxFilesAdded(maxFiles, files, files.*)'
```

Comment:
I believe you can listen for files.length here instead, haven't had the case for it before so I'm not 100% but I know you have access to a lot of properties like that.